### PR TITLE
Fix: Minimum days for vacation

### PR DESCRIPTION
**Pull request by HeitorMC**
As mentioned in #318, this PR contemplates a new error message for vacations that are less than 10 days.

image

### CODE REVIEW

File: spec/models/vacation_spec.rb [hunk 1]
```diff
@@ -72,6 +72,16 @@
 
       it { expect(vacation).to_not be_valid }
     end
+
+    context 'when the duration of the vacation is less than 10 days' do
```
Comment (adbatista) [inline spec/models/vacation_spec.rb:76]: and when vacation duration is equal 10 days and more than 10 days?

File: spec/models/vacation_spec.rb [hunk 2]
```diff
@@ -72,6 +72,28 @@
...
+
+      it { expect(vacation).to_not be_valid }
+
+      it 'raises an error' do
+        expect { vacation.save! }.to raise_error(ActiveRecord::RecordInvalid, 'A validação falhou: As férias precisam ter no mínimo 10 dias')
+      end
+    end
+
+    context 'when the duration of the vacation is equal or higher than 10 days' do
+      let(:vacation) {build(:vacation, start_date: 1.day.from_now, end_date: 11.days.from_now)}
```
Comment (adbatista) [inline spec/models/vacation_spec.rb:87]: what about exactly 10 days?

File: spec/models/vacation_spec.rb (hunk 1 above)
Comment (HeitorMC) [reply spec/models/vacation_spec.rb:76]: Done

File: spec/models/vacation_spec.rb (hunk 2 above)
Comment (HeitorMC) [reply spec/models/vacation_spec.rb:87]: Alright, now we have specs to cover all the three cases: lower than 10 days, equal to 10 days, and higher than 10 days covered

**Review by adbatista — APPROVED** [review]

**Pull request merged**